addons/l10n_pe/models/account_move.py

```diff
@@ -1,8 +1,23 @@
 # Part of Odoo. See LICENSE file for full copyright and licensing details.
 from odoo import models, fields
+from odoo.tools.sql import column_exists, create_column
 
 
 class AccountMoveLine(models.Model):
     _inherit = "account.move.line"
 
     l10n_pe_group_id = fields.Many2one("account.group", related="account_id.group_id", store=True)
+
+    def _auto_init(self):
+        """
+        Create column to stop ORM from computing it himself (too slow)
+        """
+        if not column_exists(self.env.cr, self._table, 'l10n_pe_group_id'):
+            create_column(self.env.cr, self._table, 'l10n_pe_group_id', 'int4')
+            self.env.cr.execute("""
+                UPDATE account_move_line line
+                SET l10n_pe_group_id = account.group_id
+                FROM account_account account
+                WHERE account.id = line.account_id
+            """)
+        return super()._auto_init()
```

Comment:
Can't this be automated in the ORM directly?

Reply:
Also, this field is never used... Can't we just set it to `store=False`? Or remove it? (even in stable)

This field have been *"added preventively to prepare for the EDI module"*...
https://github.com/odoo/odoo/commit/ff74a97450f551da2783164a1d195a1b0f502b9c

Reply:
I'm not sure if we can delete it: 

```
 Added related account group on the move lines to show account moves in a way Peru is used to.
```

Reply:
> Can't this be automated in the ORM directly?

It's already done, but for the simplest cases

https://github.com/odoo/odoo/blob/0aff8bb9484a23c0d875d3b12259dd4e0d51e716/odoo/fields.py#L814-L829

Reply:
How is this not a simple case?

Reply:
But it is stored and won't be changed during the installation of the module; it don't think a stored field will ever be recomputed during the installation of a module

Reply:
Yes, maybe. But this way we have to wait ORM team for review, which could take ages :grimacing:

Reply:
@rco-odoo what do you think about updating following code to add support for field like `foo_id.bar`, where bar is computed and stored 

https://github.com/odoo/odoo/blob/0aff8bb9484a23c0d875d3b12259dd4e0d51e716/odoo/fields.py#L814-L829

i.e. delete condition `not self.related_field.compute` in line 818